deployment/ccip/changeset/cs_update_rmn_config.go

```diff
@@ -0,0 +1,372 @@
+package changeset
+
+import (
+	"fmt"
+	"math/big"
+	"reflect"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+	mcmsWrappers "github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/rmn_home"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/rmn_remote"
+)
+
+type SetRMNHomeCandidateConfig struct {
```

Comment:
let's adopt the keystone idea of parameterizing even MCMS https://github.com/smartcontractkit/chainlink/blob/ca5ee25d1f49203fc7cf9fb0b6d9ad7d805ad49b/deployment/keystone/changeset/deploy_ocr3.go#L40, that will be helpful for testing/debugging as the timelock contract swallows errors and we might have environments where they are owned temporarily by the deployer key